census/tftransformestimator/README.md

```diff
@@ -0,0 +1,348 @@
+
+## Accuracy and Output
+You should see the output for default number of training steps and approx accuracy close to `80%`.
+
+# Distributed Node Training
+Distributed node training uses [Distributed TensorFlow](https://www.tensorflow.org/deploy/distributed).
+The main change to make the distributed version work is usage of [TF_CONFIG](https://cloud.google.com/ml/reference/configuration-data-structures#tf_config_environment_variable)
+environment variable. The environment variable is generated using `gcloud` and parsed to create a
+[ClusterSpec](https://www.tensorflow.org/deploy/distributed#create_a_tftrainclusterspec_to_describe_the_cluster). See the [ScaleTier](https://cloud.google.com/ml/pricing#ml_training_units_by_scale_tier) for predefined tiers
+
+## How to run the code
+You can run the code either locally or on cloud using `gcloud`.
+
+### Using gcloud local
+Run the distributed training code locally using `gcloud`.
+
+```
+export TRAIN_STEPS=1000
+DATE=`date '+%Y%m%d_%H%M%S'`
+export OUTPUT_DIR=census_distributed_$DATE
+rm -rf $OUTPUT_DIR
+```
+
+```
+gcloud ml-engine local train --module-name trainer.task \
+                             --package-path trainer \
+                             --distributed \
+                             -- \
+                             --tft-working-dir $TFT_OUTPUT_DIR_LOCAL \
+                             --train-filebase train \
+                             --eval-filebase test \
+                             --train-steps $TRAIN_STEPS \
+                             --job-dir $OUTPUT_DIR
+
+```
+
+### Using Cloud ML Engine
+Run the distributed training code on cloud using `gcloud`.
+
+```
+export SCALE_TIER=STANDARD_1
+DATE=`date '+%Y%m%d_%H%M%S'`
+export JOB_NAME=census_${SCALE_TIER}_$DATE
+export GCS_JOB_DIR=gs://<my-bucket>/path/to/my/models/$JOB_NAME
+echo $GCS_JOB_DIR
+export TRAIN_STEPS=5000
+```
+
+```
+gcloud ml-engine jobs submit training $JOB_NAME \
+                                    --stream-logs \
+                                    --runtime-version 1.4 \
+                                    --scale-tier $SCALE_TIER \
+                                    --job-dir $GCS_JOB_DIR \
+                                    --module-name trainer.task \
+                                    --package-path trainer \
+                                    --region $REGION \
+                                    --project $PROJECT \
+                                    -- \
+                                    --tft-working-dir $TFT_OUTPUT_DIR \
+                                    --train-filebase train \
+                                    --eval-filebase test \
+                                    --train-steps $TRAIN_STEPS
+
+```
+
+# Hyperparameter Tuning
+Cloud ML Engine allows you to perform Hyperparameter tuning to find out the
+most optimal hyperparameters. See [Overview of Hyperparameter Tuning](https://cloud.google.com/ml/docs/concepts/hyperparameter-tuning-overview) for more details.
```

Comment:
Please update the hyperparameter tuning docs link: https://cloud.google.com/ml-engine/docs/tensorflow/hyperparameter-tuning-overview

Reply:
Done